<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryGame</class>
 <widget class="QWidget" name="MemoryGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn00">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn02">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn03">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn01">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn11">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn13">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn21">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn22">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn23">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn30">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn31">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn32">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn33">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label00">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label03">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label01">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label02">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label13">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>105</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label21">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label22">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label23">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>205</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label30">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label31">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label32">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label33">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>305</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 10px;
margin: 3px;
background-image: url(:/images/memory-logo.png);
background-repeat: no-repeat;
background-position: center; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label30</zorder>
  <zorder>label31</zorder>
  <zorder>label32</zorder>
  <zorder>label33</zorder>
  <zorder>label23</zorder>
  <zorder>label22</zorder>
  <zorder>label21</zorder>
  <zorder>label20</zorder>
  <zorder>label13</zorder>
  <zorder>label12</zorder>
  <zorder>label11</zorder>
  <zorder>label10</zorder>
  <zorder>label03</zorder>
  <zorder>label02</zorder>
  <zorder>label01</zorder>
  <zorder>label00</zorder>
  <zorder>btn00</zorder>
  <zorder>btn02</zorder>
  <zorder>btn01</zorder>
  <zorder>btn10</zorder>
  <zorder>btn11</zorder>
  <zorder>btn12</zorder>
  <zorder>btn20</zorder>
  <zorder>btn21</zorder>
  <zorder>btn22</zorder>
  <zorder>btn30</zorder>
  <zorder>btn31</zorder>
  <zorder>btn32</zorder>
  <zorder>btn33</zorder>
  <zorder>btn23</zorder>
  <zorder>btn13</zorder>
  <zorder>btn03</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
